mlflow/server/js/src/common/utils/FeatureUtils.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This file aggregates utility functions for enabling features configured by flags.
+ * In the OSS version, you can override them in local development by manually changing the return values.
+ */
+/**
+ * UI feature preview: new ("next") version of runs comparison view within experiment runs table.
+ */
+export const shouldUseNextRunsComparisonUI = () => false;
```

Comment:
So the next-gen run comparison feature is still WIP, but there are already a few half-made views there - and it's possible to flip a "flag" right here in order to see them in OSS mlflow as well. Is it reasonable to retain it here so the OSS community will get some insight as well? Or shall we remove it from this update? WDYT @dbczumar @harupy

Reply:
@hubertzub-db I think it's fine to keep the flag here for OSS, as long as it's `false` for now :)